src/Uno.UI/UI/Xaml/Controls/Layouter/Layouter.cs

```diff
@@ -90,10 +93,14 @@ public Size Measure(Size availableSize)
 				return default;
```

Comment:
Not a bug you introduced, but I notice we don't dispose the trace activity if the element is collapsed

Reply:
We should change the pattern to use the `using var` syntax.

Reply:
Fixed